functions_framework/lib/src/function_config.dart

```diff
@@ -28,7 +28,7 @@ const _functionTypeOpt = 'signature-type';
 
 enum FunctionType {
   http,
-  cloudEvent,
+  cloudevent,
```

Comment:
You know, I really just wanted `cloudevent` originally, but figured either you or lint would complain, lol